typing/uniqueness_analysis.ml

```diff
@@ -163,12 +163,15 @@ end = struct
     | (_, access) :: _ -> access
 
   let add_barrier t uniq =
-    List.iter
-      (fun (_, access) ->
-        match access with
-        | Read barrier -> Unique_barrier.add_upper_bound uniq barrier
-        | _ -> ())
-      t
+    if Language_extension.is_at_least Unique Language_extension.Alpha
```

Comment:
Why is this one a hardcoded `Alpha` rather than `maturity_of_unique...`?

Reply:
Thanks, that should be `maturity_of_unique_for_destruction`.